### PR TITLE
add alert rules commands

### PR DESCRIPTION
**Pull request by tora-kozic**

### CODE REVIEW

File: incydr/cli/cmds/alert_rules.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        tables.append(list_as_panel(usernames[:MAX_USER_DISPLAY_COUNT]))
+        t.add_column(
+            f"{'Include' if username_filter.mode == '0' else 'Exclude'}d Usernames"
+        )
+    except MissingUsernameCriterionError as e:
+        raise e
+
+    if rule.notifications:
+        tables.append(model_as_card(rule.notifications))
+        t.add_column("Notifications")
```
Comment (tora-kozic) [inline incydr/cli/cmds/alert_rules.py:106]: Splits settings objects into adjacent columns for table output, similar to what we did for `watchlists_show`

File: incydr/cli/cmds/alert_rules.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        # Pretty sure there is only ever one "user" object whose aliases indicate the usernames on the filter
+        usernames = []
+        for user in username_filter.users:
+            usernames.extend(user.aliases)
+        tables.append(list_as_panel(usernames[:MAX_USER_DISPLAY_COUNT]))
+        t.add_column(
+            f"{'Include' if username_filter.mode == '0' else 'Exclude'}d Usernames"
+        )
+    except MissingUsernameCriterionError as e:
+        raise e
```
Comment (timabrmsn) [inline incydr/cli/cmds/alert_rules.py:102]: do we want to actually handle this?

Comment (tora-kozic) [reply incydr/cli/cmds/alert_rules.py:102]: Raising it here prevents it from going straight to the CLI logs. In terms of this specific error, the endpoint returns a `400: bad request` if the rule has no username filter, which I personally found unclear, so I don't see a real harm in calling it out specifically.

Comment (tora-kozic) [reply incydr/cli/cmds/alert_rules.py:102]: also fwiw I am planning to go through our error handling and make it more consistent/robust, currently I'm just writing custom exceptions for specific cases

**Review by timabrmsn — APPROVED** [review]

**Pull request merged**